devtools/debug.py

```diff
@@ -22,6 +22,47 @@
 StrType = str
 
 
+class DebugFrame:
```

Comment:
probably best to put this in a new module, otherwise it should go at the bottom of `debug.py`.

Reply:
I think this class is pretty tightly-coupled to all the other classes in `debug.py`, so I just moved it to the bottom.